src/Microsoft.ML.Transforms/NormalizeColumn.cs

```diff
@@ -282,6 +369,22 @@ public static NormalizeTransform Create(IHostEnvironment env, BinArguments args,
             return func;
         }
 
+        public static NormalizeTransform CreateBinningNormalizer(IHostEnvironment env, IDataView input, string labelColumn, string name, string source = null, int numBins = Defaults.NumBins, int minBinSize = Defaults.MinBinSize)
```

Comment:
>public static NormalizeTransform CreateBinningNormalizer(IHostEnvironment env, IDataView input, string labelColumn, string name, string source = null, int numBins = Defaults.NumBins, int minBinSize = Defaults.MinBinSize) [](start = 8, length = 220)

so long

Reply:
Did some formatting.

---
In reply to: [198310084](https://github.com/dotnet/machinelearning/pull/405#discussion_r198310084) [](ancestors = 198310084)